packages/core/protocols/src/proto/dxos/client/services.proto

```diff
@@ -330,9 +356,6 @@ message Invitation {
   /// Guest's identity.
   optional dxos.keys.PublicKey identity_key = 8;
 
-  /// Space to join (only present if kind is SPACE).
-  optional dxos.keys.PublicKey space_key = 9;
-
   /// Authentication code created by host (only present if auth_method is SHARED_SECRET).
```

Comment:
This should be updated to say that it's only available for the host and is excluded from encoding when invitations are shared